tests/PKSim.Tests/Infrastructure/DomainFactoryForSpecs.cs

```diff
@@ -47,6 +48,24 @@ public static Protocol CreateStandardIVProtocol()
          return protocolFactory.Create(ProtocolMode.Simple, ApplicationTypes.Intravenous).WithName("Protocol");
       }
 
+      public static Protocol CreateStandardOralProtocol()
+      {
+         var protocolFactory = IoC.Resolve<IProtocolFactory>();
+         return protocolFactory.Create(ProtocolMode.Simple, ApplicationTypes.Oral).WithName("Protocol");
+      }
+
+      public static Formulation CreateParticlesFormulation(int numberOfBins)
+      {
+         var formulationRepository = IoC.Resolve<IFormulationRepository>();
+         var formulation = formulationRepository.FormulationBy(CoreConstants.Formulation.Particles);
+         formulation.Parameter(CoreConstants.Parameters.NUMBER_OF_BINS).Value = numberOfBins;
+         
+         //set mono/polydisperse property (0=mono, 1=poly).
+         formulation.Parameter(CoreConstants.Parameters.PARTICLE_DISPERSE_SYSTEM).Value = (numberOfBins > 1) ? 1 : 0;
```

Comment:
We have constant for that
         public const int MONODISPERSE = 0;
         public const int POLYDISPERSE = 1;